lib/nostrum/api/ratelimiter.ex

```diff
@@ -47,6 +47,23 @@ defmodule Nostrum.Api.Ratelimiter do
   account for using this method.
 
 
+  ## Debugging
+
+  If you are experiencing issues with the ratelimiter, please use
+  `:sys.trace/2` to obtain debug information for the developers to be able to
+  investigate. You need to feed it the pid of the ratelimiter for your bot. For example:
+
+  ```elixir
+  Nostrum.Bot.with_bot(bot_name, fn ->
+    bot_pid = Nostrum.Bot.fetch_bot_pid()
+    ratelimiter = Nostrum.Util.get_child_pid(Nostrum.Api.Ratelimiter)
+    :sys.trace(ratelimiter, true)
+  end)
```

Comment:
`Nostrum.Util.get_child_pid/2`'s first is the supervisor pid.
Also what can be done here is either
`bot_pid = Nostrum.Bot.fetch_bot_pid()` to implicitly fetch a single running bot, or
`bot_pid = Nostrum.Bot.fetch_bot_pid(bot_name)` to explicitly fetch by name.
Then you don't have to wrap the call in `with_bot/2`